src/ansys/fluent/core/session.py

```diff
@@ -57,7 +57,8 @@ def __call__(self):
 
 
 class BaseSession:
-    """Instantiates a Fluent connection.
+    """Encapsulates a Fluent session. Expose methods for interacting with a Fluent
+    session.
```

Comment:
```suggestion
    """Encapsulates a Fluent session.
    
    This class exposes methods for interacting with a Fluent session.
```
The brief description should only consist of one fragment. All other text providing more information should be in the next paragraph.